loadtest/README.md

```diff
@@ -0,0 +1,111 @@
+# Load testing
+
+This directory contains [locust](https://locust.io) load testing tasks to load test the Global.health APIs.
+
+Set-up and initial load testing was tracked in [this github issue](https://github.com/globaldothealth/list/issues/1263).
+
+## TL;DR results
+
+Dev starts to cpu-choke at ~10 QPS (simulated with ~20 users and a spawn rate of 1), it's using the allowed 250m vCPUs fully while the RAM usage doesn't increase at all (50Mb->60Mb).
```

Comment:
Do we want to try to add an extrapolation to prod in the tl;dr? Or maybe a quick napkin calculation based on vCPUs?

Reply:
I plan on doing the extrapolation with the PR that increases the CPU available to prod yes.